scm/fake.go

```diff
@@ -137,26 +129,43 @@ func (s *FakeSCM) CreateOrgHook(ctx context.Context, opt *OrgHookOptions) error
 
 // CreateTeam implements the SCM interface.
 func (s *FakeSCM) CreateTeam(ctx context.Context, opt *TeamOptions) (*Team, error) {
-	// TODO no implementation provided yet
-	return &Team{ID: 1, Name: opt.TeamName, URL: ""}, nil
+	id := len(s.Teams) + 1
```

Comment:
id is only used once below. move code inside the uint64() conversion directly.

Reply:
Done.